lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -764,20 +765,33 @@ def test_get_index(self, attach_dataset, provide_metadata):
         indices = self.cls_obj.get_index(drb, self.index_from_metadata)
         assert indices == "unit-test.v5.result-data-sample.2020-08"
 
-    def test_exceptions_on_get_index(self, attach_dataset):
+    def test_get_index_none(self, attach_dataset):
         test = Dataset.query(name="test")
 
-        # When server index_map is None we expect 500
-        with pytest.raises(APIAbort) as exc:
-            self.cls_obj.get_index(test, self.index_from_metadata)
-        assert exc.value.http_status == HTTPStatus.INTERNAL_SERVER_ERROR
+        # No index map at all for dataset
+        assert self.cls_obj.get_index(test, self.index_from_metadata) == ""
```

Comment:
The `index_from_metadata` attribute should probably be renamed, now, eh?

Reply:
While I did consider that, it would have just made more code diffs that serve no real purpose. (Though it was an unwieldy name in the first place, and I was tempted.)